### DISCUSSION TITLE
config.toml vs. /playground/config.json welche Konfigurationsdatei ist die richtige?

### DISCUSSION THREAD

**Discussion by beautiful-orca** [martin-ueding/geo-activity-playground#172] [Q&A] [answered]
In der [Dokumentation "Using Activity Files"](https://martin-ueding.github.io/geo-activity-playground/getting-started/using-activity-files/) wird die `config.toml` Datei zum konfigurieren des Regex erwähnt.
Allerdings ist beim nutzen der Docker Version nur der `/data` Ordner im Container bzw. der gemappte playground Ordner persistent, nicht der `/app` Ordner, der diese Datei enthalten würde.
Die Konfiguration über das WebUI wird als `config.json` in den `/data` bzw. gemappten `playground` Ordner geschrieben.

Macht es Sinn die Konfiguration an einem Ort zu vereinheitlichen, also in dem `playground` Ordner?
Außerdem sollte dazu die Dokumentation klarer werden.

**Comment by martin-ueding** [comment, marked as answer]
Die TOML-Datei ist veraltet, die JSON-Datei ersetzt die komplett. Das muss in der Dokumentation aktualisiert werden. Die Einstellung der regulären Ausdrücke passiert ja jetzt auch in der Web-Oberfläche.

**Reply by beautiful-orca** [reply]
Die Beispiel Konfiguration TOML ist noch im [Verzeichnis](https://github.com/martin-ueding/geo-activity-playground/blob/master/config-example.toml).
Die braucht es dort dann auch nicht mehr?
